service/jobs/ext_api_test.go

```diff
@@ -0,0 +1,299 @@
+package jobs
+
+import (
+	"context"
+	"testing"
+
+	"github.com/databricks/databricks-sdk-go/qa"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func commonFixtureWithStatusResponse() qa.HTTPFixtures {
+	return append(
+		// API 2.2 stubs
+		[]qa.HTTPFixture{
+			{
+				Method:       "GET",
+				ReuseRequest: true,
```

Comment:
You don't need to set `ReuseRequest` on any of these mocks, since each request is called only once per client instance (i.e. per test).